app/src/main/java/com/android/sample/resources/C.kt

```diff
@@ -720,5 +720,9 @@ object C {
       const val ADD_INGREDIENT_ICON = "addIngredientIcon"
       const val NEXT_STEP_BUTTON = "nextStepButton"
     }
+
+    object Instruction {
+      const val DEFAULT_ICON = "fire"
```

Comment:
Why is the default icon a fire? Perhaps a chef's hat would be more fitting as the default icon.

Reply:
Hello @AdriB02, since we only have 4 types of icon:
```kotlin
 // IconType
    const val ICON_FIRE = "fire"
    const val ICON_SALT = "salt"
    const val ICON_MORTAR = "mortar"
    const val ICON_AXE = "axe"
```

This icon seemed the more appropriate to me. If you wish to put a different icon, don't hesitate to create one and replace it in the `C` file

Reply:
Thanks for addressing this so quickly, @akshire!